### ISSUE TITLE
Comments: How to export a blogspot blog to GitHub with C#

### ISSUE THREAD

**Issue by qwertie** [qwertie/ecsharp#2]
Associated with [this blog post](http://loyc.net/2014/linq2xml.html).

**Comment by paulrpotts** [comment]
Hello,

I am trying to use this script with LINQPad 5. I have pasted the Main() function into LINQPad, selected "C# Program," changed the filepath to point to an .xml file with a single blog post in it, and run it.

It creates the output folder, but does not write a file.

I can't set breakpoints apparently without paying for an upgrade to LINQPad "Premium." But I can use Console.Write to see what is going on.

The posts variable prints out as "(0 items)" even though the doc variable has an <entry> with title, links, author, content, etc. and doc.Root looks the same.

It looks like the line posts = doc.Root.Elements(_+"entry") is not getting anything out of the .xml files I'm using. This file was generated using a utility called "Blogger Backup" from the Blogger feed. I'm attaching a sample. Github will not let me drop an .xml file so I added a .txt extension and that seemed to work.

Any ideas what is going wrong and how to fix it?

Thanks,

Paul
[20160423T111500-Read It, Late April 2016.xml.txt](https://github.com/qwertie/ecsharp/files/240926/20160423T111500-Read.It.Late.April.2016.xml.txt)

**Comment by qwertie** [comment]
@paulrpotts I just tested my script again (copied it from this post to LINQPad) on a freshly exported blog that I never converted before. It **did** work. Make sure you are exporting your blog inside your Blogger control panel by selecting Settings \| Other \| Back Up Content.

**Comment by paulrpotts** [comment]
Hmmm, OK. I got it working with an .atom file containing a lot of posts. But it does not seem to work on a file containing just one post like the example I posted. That's what I was hoping to get working. Working on the whole .atom file produces a lot of garbage, like .html files for each spam comment on a blog post. I have found a Python script that will do what I want, but thanks for replying anyway.
